sdk/data_kusto/src/request_options.rs

```diff
@@ -0,0 +1,67 @@
+use azure_core::{AddAsHeader, HttpHeaderError, Request};
+use http::request::Builder;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct Accept<'a>(&'a str);
```

Comment:
This file seems a bit sus,
Accept and AcceptEncoding are really common headers, it's a bit weird that we need to define them like that in the kusto package specfically.
It seems like something that should be defined in azure_core.

Reply:
moved the request options to core crate